DOCSP-31847: use a session with the original client (#749)
mongodb/docs-node: source/whats-new.txt

   - The ClientEncryption API returns promises instead of callbacks.

>    - If you pass a session that was started on one
>      client to a different client, the driver throws an error when you

Reviewer: I: Passive voice and slightly ambiguous

```suggestion
   - If you start a session on client, then pass that session to a
     different client, the driver throws an error when you
```
